app/controllers/katello/api/rhsm/candlepin_proxies_controller.rb

```diff
@@ -169,17 +171,16 @@ def enabled_repos # rubocop:disable Metrics/MethodLength
         if !repo['baseurl'].blank?
           URI(repo['baseurl'].first).path
         else
-          logger.warn("System #{@system.name} (#{@system.id}) attempted to bind to unspecific repo (#{repo}).")
+          logger.warn("System #{@host.name} (#{@host.id}) attempted to bind to unspecific repo (#{repo}).")
           nil
         end
       end
 
-      processed_ids, error_ids = @system.save_bound_repos_by_path!(paths.compact)
-
-      result = {:processed_ids => processed_ids,
-                :error_ids     => error_ids,
-                :result        => "ok"}
-      result[:result] = "error" if error_ids.present?
+      result = nil
```

Comment:
it is, but the variable has to exist outside the block for it to exist after the block is finished executing.